yarn.lock

```diff
@@ -0,0 +1,25 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
There's already a package lock in the same place.

Reply:
I'm using the `yarn2nix` pattern, which requires `yarn.lock`. the `package-lock.json` version (called `node2nix`) has some more overhead and is less pure (requires unsandbox'd internet access)

Reply:
Should I merge as-is then?

Reply:
if you don't mind having both `yarn` and `npm` lockfiles, then it's ready.

Reply:
This repo is not going to be used for much longer, so no point being picky.